templates/controllers/grid/users/reviewer/form/reviewerFormFooter.tpl

```diff
@@ -8,13 +8,6 @@
  * The non-searching part of the add reviewer form
  *
  *}
-<script type="text/javascript">
-	$(function() {ldelim}
-		// Attach the form handler.
-		$('#reviewerFormFooter').pkpHandler('$.pkp.controllers.grid.users.reviewer.form.ReviewerFormFooterHandler');
-	{rdelim});
-</script>
-
```

Comment:
I see now why you removed the file from the minified scripts. But you need to remove the js file also. I think I didn't see the removal.